rcl_yaml_param_parser/README.md

```diff
@@ -1,6 +1,15 @@
 **ROS2 rcl YAML paramter parser**
 
-Parse a YAML parameter file and populate the C data structure
+Parse a YAML parameter file and populate the C data structure.
```

Comment:
With the other packages linking to docs.ros2.org, I think you should add this README to the doxygen file as well or mimic how they put the features in the source file.